### PR TITLE
issue 376 -- parse and ignore RENAME INDEX statements

### PR DESCRIPTION
**Pull request by osheroff**
https://github.com/zendesk/maxwell/issues/376

@zendesk/rules

### CODE REVIEW

File: src/test/java/com/zendesk/maxwell/schema/ddl/DDLParserTest.java [hunk 1]
```diff
@@ -200,7 +200,8 @@ public void testParsingSomeAlters() {
 			"create table `shard1.foo` ( `id.foo` int ) collate = `utf8_bin`",
 			"ALTER TABLE .`users` CHANGE COLUMN `password` `password` VARCHAR(60) CHARACTER SET 'utf8' COLLATE 'utf8_bin' NULL DEFAULT NULL COMMENT 'Length 60 for Bcrypt'",
 			"create table `shard1.foo` ( `id.foo` int ) collate = `utf8_bin`",
-			"create table if not exists audit_payer_bank_details (event_time TIMESTAMP default CURRENT_TIMESTAMP())"
+			"create table if not exists audit_payer_bank_details (event_time TIMESTAMP default CURRENT_TIMESTAMP())",
+			"ALTER TABLE foo RENAME INDEX index_quote_request_follow_on_data_on_model_name TO index_quote_request_follow_on_data_on_model_class_name"
```
Comment (dadah89) [inline src/test/java/com/zendesk/maxwell/schema/ddl/DDLParserTest.java]: oh so this is possible b/c it is just parsing testing, not being executed against mysql 5.5 and 5.6

File: src/main/antlr4/imports/mysql_alter_table.g4 [hunk 2]
```diff
@@ -62,6 +62,7 @@ ignored_alter_specifications:
      */
      | ALGORITHM '='? algorithm_type
      | LOCK '='? lock_type
+     | RENAME (INDEX|KEY) name TO name
```
Comment (dadah89) [inline src/main/antlr4/imports/mysql_alter_table.g4]: wondering if it'd be useful to indicate 5.5/5.6/5.7 specific syntax, e.g. dropping 5.5 support

**Comment by dadah89** [comment]
👍

File: src/main/antlr4/imports/mysql_alter_table.g4 (hunk 2 above)
Comment (osheroff) [inline src/main/antlr4/imports/mysql_alter_table.g4]: not sure what you mean.

Comment (dadah89) [inline src/main/antlr4/imports/mysql_alter_table.g4]: nvm, I thought there were more syntax changes between versions

**Pull request merged**